fabric-dimensions-v1/src/build.gradle

```diff
@@ -0,0 +1,6 @@
+archivesBaseName = "fabric-dimensions-v0"
```

Comment:
```suggestion
archivesBaseName = "fabric-dimensions-v1"
```

Reply:
On this the packaging seems to be wrong as well.